CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.22.1] - 2021-09-13
+- Add extensions directory to the resource root.
```

Comment:
This isn't quite accurate. It's not "adding the directory to the resource root", it's marking the directory _as a_ resource root. A given source set has multiple resource roots (directories that are treated as the root directory containing resources to be copied over in the build/publish process). By default, only `resources` it treated as a resource root for a given source set, but here we're also adding `pegasus` and `extensions` (or whatever the user configures) as extra resource roots.

I would also clarify that this is done by the Gradle plugin (I doubt that it will affect anyone, but it's best to keep things accurate and specific).

Reply:
Updated